docs/tutorials/timeseries/forecasting-chronos.ipynb

```diff
@@ -9,13 +9,19 @@
     "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/autogluon/autogluon/blob/master/docs/tutorials/timeseries/forecasting-chronos.ipynb)\n",
```

Comment:
<p>new, more accurate, and up to 250x faster than the original Chronos models. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/autogluon/autogluon/pull/4684/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>